src/main/java/org/concordion/api/AbstractCommand.java

```diff
@@ -25,7 +25,4 @@ public boolean isExample() {
 
     public void executeAsExample(CommandCall commandCall, Evaluator evaluator, ResultRecorder resultRecorder) {
     }
-
-    public void finish(CommandCall commandCall) {

```

Comment:
This is an API breaking change - there might be some commands in extensions that rely on finish being called at an appropriate time.